src/io/flutter/FlutterBundle.properties

```diff
@@ -211,3 +211,16 @@ settings.enable.embedding.devtools=Enable embedding DevTools in the Flutter Insp
 action.new.project.title=New Flutter Project...
 welcome.new.project.title=Create New Flutter Project
 welcome.new.project.compact=New Flutter Project
+
+npw_platform_android=Android
+npw_platform_ios=iOS
+npw_platform_linux=Linux
+npw_platform_macos=MacOS
+npw_platform_web=Web
+npw_platform_windows=Windows
+npw_platform_availability_help=Platform availability depends on the Flutter SDK channel, and on 'flutter config'.
```

Comment:
I wasn't sure how verbose we should be. I considered adding another line with some explanation, and think I'll do that now. However, I misunderstood the purpose of the config file. I'm going to have to rewrite some of this. Instead of enabling check boxes based on platforms enabled in the config file it seems those should be the ones checked by default. All should be enabled.